src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background: $color-bg-white;
+  width: $card-width;
+  height: $card-height;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-sizing: border-box;
+  position: relative;
+  padding: 16px;
+
+  &__image {
+    display: block;
+    background-image: url(../images/imac.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 20px;
+  }
+
+  &__title {
+    color: $color-text-dark;
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin-top: 20px;
+  }
+
+  &__product-code {
+    font-size: $font-size-small;
+    font-weight: 400;
+    line-height: $font-size-large;
```

Comment:
The line-height for `&__product-code` seems to be set to `$font-size-large`, which might not be appropriate. Line height usually doesn't match font size directly and should be greater than the font size for better readability. Please check if this is intentional or a mistake.